.github/workflows/build.yml

```diff
@@ -35,7 +37,6 @@ jobs:
       GOFLAGS: -mod=vendor
       SHELL: bash
       GITHUB_REPO_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-    timeout-minutes: 30
```

Comment:
Instead of specifying timeout here, do so in the integration test runners that would be responsible for such a timeout. Manual integration test runs have a 10m timeout, and automated nightly integration test runs have a 90m timeout.